client/ios/migration-guide.mdx

```diff
@@ -0,0 +1,164 @@
+---
+
+title: "RTVIClient Migration Guide for IOS"
+sidebarTitle: "Migration Guide"
+description: "A Guide to migrating from RTVIClient to PipecatClient for iOS"
+----------------------------------------------------------------------------
+
+This guide covers migrating from the `RTVIClient` to the new `PipecatClient` in an iOS (Swift) application. The new client introduces simplified configuration, modern messaging patterns, and improved function call handling. For an overview of the changes, see the top-level [RTVIClient Migration Guide](https://docs.pipecat.ai/client/migration-guide).
+
+## Key Changes
+
+### 1. Package and Class Names
+
+**Old**
+
+```swift
+import RTVIClient
+import RTVIClientIOSDaily
+```
+
+**New**
+
+```swift
+import PipecatClientIOS
+import PipecatClientIOSDaily
+```
+
+### 2. Client and Transport Configuration
+
+Previously, the client constructor accepted parameters like pipeline or endpoint configuration. Now, the configuration is passed explicitly to the `startBotAndConnect()` or `connect()` methods, and the constructor takes only transport options.
+
+**Old**
+
+```swift
+let client = RTVIClient(params: ...)
+```
+
+**New**
+
+```swift
+let pipecatClientOptions = PipecatClientOptions(
+    transport: DailyTransport(),
+    enableMic: true,
+    enableCam: false
+)
+
+let client = PipecatClient(options: pipecatClientOptions)
+```
+
+### 3. Connection Method
+
+The connection process is now explicit and supports modern async handling.
+
+**Old**
+
+```swift
+client.connect()
+```
+
+**New**
+
+```swift
+let startBotParams = APIRequest(endpoint: URL(string: "https://your-server/connect")!)
+client.startBotAndConnect(startBotParams: startBotParams) { result in
+    switch result {
+    case .success(let connectionParams):
+        print("Connected successfully: \(connectionParams)")
+    case .failure(let error):
+        print("Connection failed: \(error)")
+    }
+}
+```
+
+### 4. Function Call Handling
+
+Helpers like `LLMHelper` have been removed. You now register function call handlers directly on the `PipecatClient`.
+
+**Old**
+
+```swift
+let llmHelper = LLMHelper()
+client.registerHelper("llm", llmHelper)
```

Comment:
can this be expanded to show whatever was previously required to specifically handle a function call? I want to make it clear that `registerFunctionCallHandler` only replaces that specific functionality and not all of what `llmHelper` could do before.

Reply:
Done.